package.json

```diff
@@ -13,11 +13,13 @@
     "mocha": "^4.0.1"
   },
   "scripts": {
-    "test": "npm run node-tests && npm run ember-tests",
+    "test": "npm run node-tests && npm run ember-tests && npm run truffle-tests",
     "node-tests": "mocha ./packages/test-support/bin/run.js --timeout 10000",
     "node-lint": "mocha ./packages/test-support/bin/lint.js",
     "ember-tests": "node packages/test-support/ember-test-runner.js",
-    "server": "node packages/hub/bin/server.js"
+    "truffle-tests": "LOG_LEVELS='*=warn' node packages/test-support/truffle-test-runner.js",
+    "server": "node packages/hub/bin/server.js",
+    "blockchain": "./node_modules/.bin/truffle develop"
```

Comment:
I think you don't need `./node_modules/.bin` here. Everything inside these scripts runs with that on the PATH already.

Reply:
👍